components/widgets/resources/src/main/resources/resources/uicomponents/widgets/exportData.css

```diff
@@ -110,7 +110,6 @@
 }
 .xdialog-box .question {
```

Comment:
as discussed, i would use padding in addition to min-width, with long-text buttons it becomes too condensed again, IMHO. Also maybe  a bit wider than 5em? I tried and the following seems to look good:

    min-width: 6em;
    padding-right: 1em;
    padding-left: 1em;